evaluate_library_access_from_output_tsv/read_and_merge_library_access_datasets.R

```diff
@@ -0,0 +1,45 @@
+#### Read the datasets ####
+
+read_and_merge_library_access_datasets <- function(
```

Comment:
Let's use `readr::read_tsv`, which should infer the compression properly

Reply:
This is now implemented in d2b46814.